lib/Connection.php

```diff
@@ -21,6 +21,17 @@
 {
 
 	/**
+	 * The DateTime format to use when translating other DateTime-compatible objects.
+	 *
+	 * NOTE!: The DateTime "format" used must not include a time-zone (name, abbreviation, etc) or offset.
+	 * Including one will cause PHP to ignore the passed in time-zone in the 3rd argument.
+	 * See bug: https://bugs.php.net/bug.php?id=61022
+	 *
+	 * @var string
+	 */
+	const DATETIME_TRANSLATE_FORMAT = 'Y-m-d\TH:i:s';
```

Comment:
It's just the RFC 3339 format without the timezone identifier. I figured it was the most easily (standardized, spec-based) identifiable format. It doesn't really matter, though, as it's just for internal translations (not for storage, etc).